pkg/reconciler/backup/volumesnapshot/reconciler.go

```diff
@@ -585,11 +585,11 @@ func (se *Reconciler) waitSnapshotToBeReady(
 }
 
 // getSnapshotName gets the snapshot name for a certain PVC
-func getSnapshotName(backupName string, role utils.PVCRole) (string, error) {
```

Comment:
This function can be moved inside `pkg/reconciler/persistentvolumeclaim/pvc_role.go`, as another member of the `PVCRole` interface. Something like:

```
GetSnapshotName(backupName string) string
```

Reply:
done